tf_yarn/pytorch/model_ckpt.py

```diff
@@ -57,10 +55,7 @@ def load_ckpt(
 def save_ckpt(
     model_dir: str, model: Union[DDP, torch.nn.Module], optimizer: torch.optim.Optimizer,
     epoch: int, **kwargs: Dict[Any, Any]
-) -> Optional[str]:
-    if int(os.environ[PYTORCH_DPP_RANK]) != 0:
```

Comment:
This will break our code to train CLIP on yarn. Can we have this feature optional ? Like:
if "PYTORCH_DPP_RANK" in os.environ and int(os.environ[PYTORCH_DPP_RANK]) != 0:
    return None